### PR TITLE
[cisco_ftd] Remove Invalid ID

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Removal of the value `Invalid ID`

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -2037,7 +2037,7 @@ processors:
   # Define ECS Host and Device fields from Cisco ISE pxGrid Endpoint Profile data
   - remove:
         field: _temp_.host.type
-        if: "ctx._temp_?.host?.type != null && ctx.host?.type == 'Invalid ID'"
+        if: "ctx._temp_?.host?.type != null && ctx._temp_?.host?.type == 'Invalid ID'"
```
Comment (efd6) [inline packages/cisco_ftd/data_stream/log/elasticsearch/ingest_pipeline/default.yml:2040]:
```suggestion
        if: "ctx._temp_?.host?.type != null && ctx._temp_.host.type == 'Invalid ID'"
```

**Comment by efd6** [comment]
Is there a particular problem that this solves?

**Comment by LaZyDK** [comment]
It just removes a string that has no value. I intended to have this work when I initially did the PR, but missed a field.

**Review by efd6 — APPROVED** [review]

**Comment by efd6** [comment]
Need to run `elastic-package build`.

**Comment by LaZyDK** [comment]
That's better.

**Pull request merged**